### DISCUSSION TITLE
Github Student Rejected as I am not near my school for Summer Break

### DISCUSSION THREAD

**Discussion by yfz3357** [community/community#129265] [GitHub Education]
### Hi everyone,

As titled, my github student application is rejected due to I traveled back home for summer break. What are some ways to resolve this?

**Comment by davevad93** [comment]
Hi @yfz3357 ,

Take a look at this post: [Student Developer Pack Application FAQs & Common Rejection Reasons (Updated!)](https://github.com/orgs/community/discussions/111352)

Search for the "`You appear to not be near any campus location for the school you have selected.`" section.

There's a workaround for distance learning student, but I don't know if there's an effective solution for those who are just on summer break far from school.

**Reply by ericagb** [reply]
Indeed there seems to be no solution for students on summer break. This is a bummer. There should be some sort of manual approval process.

**Reply by davevad93** [reply]
You're right @ericagb, but I doubt they're going to do that since they've also stopped doing individual manual reviews or status updates regarding GitHub Education support tickets. I suppose that given the high volume of requests they get on a daily basis, it would be a near impossible thing to do, still it would be nice if there was some sort of manual review process available for special cases such as this one.

**Reply by yfz3357** [reply]
RIP :(
